ledger/metrics/src/test/suite/scala/com/daml/metrics/InstrumentedSourceSpec.scala

```diff
@@ -21,52 +21,15 @@ final class InstrumentedSourceSpec extends AsyncFlatSpec with Matchers with Akka
 
   behavior of "InstrumentedSource.queue"
 
-  it should "correctly enqueue and track the buffer saturation" in {
```

Comment:
Is this testcase no longer valid?

Reply:
This test case is covered below for the `BoundedSourceQueue`. We no longer support `OverflowStrategy.backpressure` and others.